photon-core/src/main/java/org/photonvision/vision/camera/LibcameraGpuSettables.java

```diff
@@ -133,23 +134,29 @@ public void setExposure(double exposure) {
             return;
         }
 
+        // Map exposure from 0 to 100 (UI Values) to 0.1 to 100 since 0 in libcamera is auto exposure.
+        // 0.1 since it is the next lowest value available through the UI.
+        MathUtils.map(exposure, 0.0, 100.0, 0.1, 100);
+
         // HACKS!
         // If we set exposure too low, libcamera crashes or slows down
         // Very weird and smelly
         // For now, band-aid this by just not setting it lower than the "it breaks" limit
-        //  is different depending on camera.
+        // is different depending on camera.
         if (sensorModel == LibCameraJNI.SensorModel.OV9281) {
-            if (exposure < 6.0) {
-                exposure = 6.0;
-            }
+            // Map and clamp exposure to 6 to 100 since setting exposure on ov9281 lower
+            // than 6 will crash libcamera at resolutions lower than max.
+            exposure = MathUtils.map(exposure, 0.1, 100.0, 6.0, 100);
```

Comment:
These magic numbers rely on the 8000 number below (that concerts exposure percent totime in uS) to not change right? Is there a way to do this that more directly maps to that magic number?